app/javascript/dashboard/routes/dashboard/settings/sla/EditSLA.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="h-auto overflow-auto flex flex-col">
+    <woot-modal-header :header-title="pageTitle" />
+    <sla-form
+      :submit-label="$t('SLA.FORM.EDIT')"
+      :selected-response="selectedResponse"
+      @submit="editSLA"
+      @close="onClose"
+    />
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+import alertMixin from 'shared/mixins/alertMixin';
+import validationMixin from './validationMixin';
+import validations from './validations';
+import SlaForm from './SlaForm.vue';
+
+export default {
+  components: {
+    SlaForm,
+  },
+  mixins: [alertMixin, validationMixin],
+  props: {
+    selectedResponse: {
+      type: Object,
+      default: () => {},
+    },
+  },
+  data() {
```

Comment:
Please check if the data is really required here.

Reply:
Thank you for catching that. Removed.